examples/hbase-snapshot-migration-scripts/restore-snapshot.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+# This script restores snapshots on the cluster
+# Run this script on the master node of HBase 2x cluster
+# Usage: ./restore-snapshot.sh <namespace>
+
+# Parse the argument
+if [ -z "$1" ] || [ "$1" == '-h' ]; then
+  echo "Usage: $0 <namespace>"
+  exit 1
+fi
+
+# Create the namespace for tables
+mapfile -t arr < <(echo "list_namespace" | hbase shell -n | sed '2,${/seconds/d;}' | sed '2,${/row(s)/d;}' | sed '1d')
+for i in "${arr[@]}"; do
+  if [ "${arr[$i]}" == "$1" ]; then
+    echo "Namespace ${arr[$i]} already exists"
+  else
+    echo "create_namespace '$1'" | hbase shell -n
+    # Check if the namespace is created
+    echo "list_namespace" | hbase shell -n
+    break
+  fi
+done
```

Comment:
What if the target namespace is in the list but isn't first in the list? Would we try creating the namespace and fail?

Reply:
yes that was happening. updated the code so display namespace exists instead of a java exception.